modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/utils/ellipsize.es.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export default (value, lenght) => value.substring(0, lenght) + '...';
```

Comment:
@alinedoleron, ideally, this code should be committed inside [this commit](https://github.com/natocesarrego/liferay-portal/pull/628/commits/136e560ef448a62f60e78905f6f83ab6c7efff5e) or be added before this mentioned commit, because the `ellipsize` is being invoked before its declaration and it can affect the reviewer's understanding. 😉 This is not a big deal, just keep it in mind for next pulls, ok?

Reply:
ok! Thank you.